compliance/aws/disallowed_regions/README.md

```diff
@@ -0,0 +1,54 @@
+##  AWS Disallowed Regions
+
+### What it does
+
+This policy checks all instances in a set of disallowed regions. The user is given the option to Terminate the instance after approval.
+
+### Functional Details
+
+- The policy leverages the AWS API to check all instances that exist in a disallowed region.
+- When an EC2 instances in disallowed region is detected, an email action is triggered automatically to notify the specified users of the incident. Users then have the option to terminate instances after manual approval if needed.
```

Comment:
Change **an EC2 instances** to **an EC2 instance**

Reply:
Done changes as per comments